test/testpaint/TestTrack.cpp

```diff
@@ -166,6 +166,9 @@ class EntranceStyleFilter : public ITestTrackFilter {
                          Ride *ride, rct_ride_entry *rideEntry
     ) override {
         ride->entrance_style = variant;
+
+        Ride * rct2ride = RCT2_ADDRESS(0x013628F8, Ride);
```

Comment:
Perhaps you could use `rct2_ride` in TestPaint?

Reply:
Will do.